### ISSUE TITLE
Update to Node 12

### ISSUE THREAD

**Issue by patjouk** [mozilla/donate-wagtail#996]
While updating dependencies today, I discovered that we're still running on Node 8, which is now EoL. We should update to Node 12 that is the active LTS.

**Comment by Pomax** [comment]
We may want to jump to 14, rather than 12, as that became the current LTS (in parallel with 12) at the end of April

**Comment by patjouk** [comment]
I don't agree it's 1 as an estimate. Plus, if we want to move to Node 14, we should close that issue, open an epic and migrate all apps to 14 to avoid repeat the same mistake we did here.
